src/benchmarks/micro/libraries/System.IO/StreamReaderReadToEndTests.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.IO.Tests;
```

Comment:
Please excuse me for not documenting it anywhere, but we can't use latest C# features as this we sometimes use old (3.1) SDK to compile all the benchmarks and compare .NET 3.1 vs 5 etc. Please use classic namespaces.

Here is what I had to do recently to get it working for 3.1: https://github.com/dotnet/performance/pull/2235